server.js

```diff
@@ -0,0 +1,100 @@
+const express = require("express");
+const ramenData = require("./ramenData")
+const path = require('path')
```

Comment:
typically when importing, packages go on the very top and local files go below that, separated by a space
```suggestion
const express = require("express");
const path = require('path')

const ramenData = require("./ramenData")
```